src/components/text-editor/text-editor.tsx

```diff
@@ -229,8 +233,12 @@ export class TextEditor implements FormComponent<string> {
         }
     };
 
-    private handleChange = () => (event: CustomEvent<string>) => {
+    private handleChange = (event: CustomEvent<string>) => {
         event.stopPropagation();
-        this.change.emit(event.detail);
+        this.changeEmitter(event.detail);
     };
+
+    private changeEmitter = debounce((value: string) => {
+        this.change.emit(value);
+    }, DEBOUNCE_TIMEOUT);
```

Comment:
Since we've had problems with limel-input-field which we think are _caused_ by the debounce, it's feels a little worrying that we are here _adding_ debounce to get rid of what sounds like very similar problems… 🤔

I'm not saying I have any ideas on how else to solve it, I'm just worried that the underlying problem here is a race-condition or something, and that debounce is really just a band-aid rather than a proper fix.

I'm not sure if I'm right, but my _instinct_ says that debounce is not a fix for anything but performance issues. If things aren't working properly (apart from how fast it is) without the debounce, adding debounce is probably just hiding the problem by making it occur less often, or much less likely to occur.

I'd be happy to have a look at this, if you'd like me to.

Reply:
I would like that a lot, if you find the time. I agree that debounce isn't really tackling the issue fundamentally. But should we consider adding the debounce as a temporary fix? Add a `// TODO`?

Reply:
TODO's never get addressed… 😞

I'll see if I can take a stab at it, but yeah, if this is working, you don't have to wait for that 👍